helper.py

```diff
@@ -0,0 +1,38 @@
+"""
+
+Copies files and folders to relevant containers
+"""
+import subprocess
+import os
+
+def cp_helper(name, test_name):
```

Comment:
I don't like very much that you put `jupyter-` in different places. Can you create a `get_container_name` method and call it? This is me being picky.